### ISSUE TITLE
framed pictures with transparent or white background

### ISSUE THREAD

**Issue by asiasowa** [BradyAJohnston/figpatch#2]
Hi, 

this package is great! Thanks for developing and sharing it :) 
I'm experiencing some issues - I was loading .png files with transparent background and it automatically loaded with a gray background.
Then, I changed the background of my files to white, but I still have some frame/shadow, even though  I try to avoid it with additional options (  fig("filepath", aspect.ratio = "free", b_pos = "inside", b_size = 0, b_margin =  ggplot2::margin(0, 0, 0, 0)) ). I'm attaching  the example. b2 and c2 represent .png files loaded with fig() function. 

![Picture1](https://user-images.githubusercontent.com/18326852/138978896-5305889b-2b79-4007-a277-254f7aef0720.png)

I would be very grateful for any suggestions :) 
Thanks in advance!

**Comment by BradyAJohnston** [comment]
Hello! Thanks so much for the bug report. Glad to see that it is being useful for people. 

I have just fixed the problem with it in the dev version, so if you install the dev version from github it should work work :) 

```r
#install.packages("devtools")
devtools::install_github("bradyajohnston/figpatch")
```

If interested, it turns out I hadn't cleared out the default theme properly in the `fig()` function, so it was drawing a gray rectangle behind the transparent png borders. If you add `fig() + theme(panel.background = element_blank())` it should be a quick fix as well without install the dev version.

Let me know if this fixes your problem, and it's great to see it in action! If it works I'll push this update with a few other things to CRAN later in the week.

**Comment by asiasowa** [comment]
Thanks for a quick response and action :) Just installed the dev version and it works!!!! 😃

**Comment by BradyAJohnston** [comment]
Finally got around to updating on CRAN, is now fixed in 0.2.
